gcloud/bigtable/happybase/connection.py

```diff
@@ -265,6 +269,77 @@ def tables(self):
 
         return table_names
 
+    def create_table(self, name, families):
+        """Create a table.
+
+        .. warning::
+
+            The only column family options from HappyBase that are able to be
+            used with Cloud Bigtable are ``max_versions`` and ``time_to_live``.
+
+        .. note::
+
+            This method is **not** atomic. The Cloud Bigtable API separates
```

Comment:
@jgeewax Who should we reach out to about this question? HBase/Thrift allow the columns to created when the table is, but [`CreateTableRequest`][1] does not.

To deal with it here we could also try to clean up, e.g. if there is any failure during creation, we try to delete all the things that were created. But IMO that is opening Pandora's box.

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/7c77c170c8818a0440e0d2f4ecedcb8329c1f8d9/bigtable-protos/src/main/proto/google/bigtable/admin/table/v1/bigtable_table_service_messages.proto#L26